nifi-extension-bundles/nifi-slack-bundle/nifi-slack-processors/src/main/java/org/apache/nifi/processors/slack/ListenSlack.java

```diff
@@ -96,6 +98,8 @@ public class ListenSlack extends AbstractProcessor {
         "The Processor is to receive only slack messages that mention the bot user (App Mention Events)");
     static final AllowableValue RECEIVE_COMMANDS = new AllowableValue("Receive Commands", "Receive Commands",
         "The Processor is to receive Commands from Slack that are specific to your application. The Processor will not receive Message Events.");
+    static final AllowableValue MEMBER_JOINED_CHANNEL_EVENTS = new AllowableValue("Member Joined Channel", "Member Joined Channel",
```

Comment:
The pattern of the other values seems to be `Receive...` so from that perspective, should this be called `Receive Joined Channel Events`?

Reply:
Done, thanks @exceptionfactory !